index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portofolio</title>
+</head>
+<body>
+  <div class="container">
+    <div class="toolbar">
```

Comment:
- [ ] I think it would be better if you wrap your toolbar in a `header` tag instead of a `div` as the` header` element represents the introductory content of a webpage just like yours which will make your website semantic and also good for SEO. This will also enable screen readers to understand that it's the introductory content 😄 . 

Checkout this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more details